mibidata/tiff.py

```diff
@@ -90,6 +94,24 @@ def write(filename, image, sed=None, optical=None, ranges=None,
                          'instance.')
     if image.coordinates is None or image.size is None:
         raise ValueError('Image coordinates and size must not be None.')
+    if image.fov_id is None:
```

Comment:
Since you've already added `REQUIRED_METADATA_ATTRIBUTES` above, I think you could simplify all of this by looping through each and checking if None with `getattr()`. In that case, it's fine to print out an error message containing the attribute name instead of a more readable version, e.g. `fov_name is required and may not be None`.

It might be even better to collect a list of all of the attributes that are None and concat the list to print a single error message. That way a user doesn't have to go through and set each one before finding out the next one is required, having to get 10 ValueErrors before succeeding. For example: `fov_name, run, and folder are required and may not be None`.

Reply:
> Since you've already added REQUIRED_METADATA_ATTRIBUTES above, I think you could simplify all of this by looping through each and checking if None with` getattr()`
I now use the list of required attributes and loop through

> It might be even better to collect a list of all of the attributes that are None and concat the list to print a single error message.
fixed so error message lists all missing metadata fields